contentwarehouse/snippets/fetch_acl_sample.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_fetch_acl]
+
+from google.cloud import contentwarehouse
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_number = 'YOUR_PROJECT_NUMBER'
+# location = 'YOUR_PROJECT_LOCATION' # Format is 'us' or 'eu'
+# document_id = 'YOUR_DOCUMENT_ID'
+# user_id = 'YOUR_SERVICE_ACCOUNT' # Format is "user:xxxx@example.com"
+
+
+def fetch_acl(
+    project_number: str,
+    location: str,
+    user_id: str,
+    document_id: str = ''
+) -> None:
```

Comment:
Please add a docstring explaining what this method does. Even if it's obvious from the name, describe the arguments and maybe provide example values.